development/development.env

```diff
@@ -124,3 +124,6 @@ NAUTOBOT_SSOT_LIBRENMS_HOSTNAME_FIELD="sysName" # hostname or sysName
 
 NAUTOBOT_SSOT_ENABLE_SOLARWINDS="False"
 
+NAUTOBOT_SSOT_ENABLE_VSPHERE="True"
```

Comment:
We require all integrations except for Itential to pass CI without the integration enabled unless you have a Jobs test that requires the Job be installed.

Reply:
Thank you! Updated!